eva/parser/eva.lark

```diff
@@ -20,7 +20,7 @@ create_database: CREATE DATABASE if_not_exists? uid
 
 create_index: CREATE INDEX uid ON table_name index_elem vector_store_type?
 
-create_table: CREATE TABLE if_not_exists? table_name create_definitions 
+create_table: CREATE TABLE if_not_exists? table_name (create_definitions | (AS select_statement))
```

Comment:
Do we plan to not support `CREATE TABLE if_not_exists? table_name create_definitions AS select_statement`? So we can still rename the column. It is fine that we do not include in this pr. Check the plan here.

Reply:
Ideally yes.